src/playlist/Length.cxx

```diff
@@ -36,15 +36,16 @@ playlist_provider_length(Response &r,
 
 	std::unique_ptr<DetachedSong> song;
 	unsigned i = 0;
-	SignedSongTime playtime = (SignedSongTime)0;
+	std::chrono::milliseconds playtime = (std::chrono::milliseconds)0;
 	while ((song = e.NextSong()) != nullptr) {
 		if (playlist_check_translate_song(*song, base_uri,
 						  loader))
 			playtime += get_duration(*song);
 		i++;
 	}
 	r.Fmt(FMT_STRING("songs: {}\n"), i);
-	r.Fmt(FMT_STRING("playtime: {}\n"), playtime.RoundS());
+	const auto seconds = std::chrono::duration_cast<std::chrono::seconds>(playtime);
+	r.Fmt(FMT_STRING("playtime: {}\n"), seconds.count());
```

Comment:
This removes the rounding which was very explicit previously

Reply:
Replaced with std::chrono::round